misk/src/main/kotlin/misk/security/ssl/TlsProtocols.kt

```diff
@@ -3,4 +3,6 @@ package misk.security.ssl
 object TlsProtocols {
   /** List of safe to use TLS protocols, in preferred order */
   val safe = arrayOf("TLSv1.2", "TLSv1.1")
+  /** List of modern TLS protocols for extra security, in preferred order */
+  val modern = arrayOf("TLSv1.3")
```

Comment:
I really like Google Cloud’s terms for these. In their terminology, `safe` would be `modern`. And maybe rename this `restricted` and add TLSv1.2 ? 

https://cloud.google.com/load-balancing/docs/ssl-policies-concepts